Sources/HTTPParser/http_parser.swift

```diff
@@ -176,7 +312,7 @@ private let F_SKIPBODY: http_flags              = 1 << 6
 private let F_CONTENTLENGTH: http_flags         = 1 << 7
 
 /* Map for errno-related constants */
-public enum http_errno: String, Error {
+public enum http_errno: StaticString, Error {
```

Comment:
The following function uses the String (StaticString) to provide the name of the error via .rawValue:
```
/* Return a string name of the given error */
public class func errno_name(_ err: http_errno) -> String {
  return err.rawValue
}
```
This function is not used in bench.c/.swift so I thought it would not effect performance.  However I replaced StaticString with String and the benchmark did slow down.  More investigation is required.